src/main/java/com/conveyal/gtfs/loader/JdbcTableWriter.java

```diff
@@ -755,8 +755,9 @@ private int updateStopTimesForPatternStop(ObjectNode patternStop, int previousTr
      *
      * TODO? add param Set<String> serviceIdFilters service_id values to filter trips on
      */
-    private int updateStopTimesForPatternStops(List<PatternStop> patternStops) throws SQLException {
+    private int updateStopTimesForPatternStops(List<PatternStop> patternStops, boolean interpolateStopTimes) throws SQLException {
         PatternStop firstPatternStop = patternStops.iterator().next();
```

Comment:
(Unlikely edge case) Say I accidentally make a pattern with zero stops and click `Normalize stop times` it throws a NoSuchElemenException and a long error. 
![image](https://github.com/conveyal/gtfs-lib/assets/62163307/5bc07a0a-9ac4-490d-ab23-493ee8241b1a)
Is there a way to check for an empty list and to return a cleaner error message?

Reply:
I think this is best addressed by preventing the behaviour client side, although I'll defer to other reviewers if they'd like some additional checks